cmd/clair/main.go

```diff
@@ -132,10 +132,22 @@ func Boot(config *Config) {
 	st := stopper.NewStopper()
 
 	// Open database
-	db, err := database.Open(config.Database)
-	if err != nil {
-		log.Fatal(err)
+	var db database.Datastore
+	var dbError error
+	maxConnectionAttempts := 20
```

Comment:
can we make this a constant at the top of the file and rename it `maxDBConnectionAttempts`?